app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Post;
+use Auth;
+
+class PostController extends Controller
+{ 
+    public function createPost(Request $request)
```

Comment:
convention would be call this method 

`public function store(Request $request)`

Reply:
I'll make sure to remember that!

Reply:
See Resource controllers in the Laravel docs; this is what a lot of controllers end up looking like.

Reply:
I checked out the documentation just now and it made a lot of sense. I'll make sure to follow that convention as well :) Thank you for pointing that out